sceptre/resolvers/project_variables.py

```diff
@@ -24,6 +26,12 @@ def resolve(self):
         :returns: Value at argument location.
         :rtype: str
         """
+        init(autoreset=True)
+        simplefilter('always', DeprecationWarning)
+        warn(Fore.YELLOW + (
+                "The project_variables resolver is being deprecated, and will "
```

Comment:
I tried that and the lint complained about it. This is the only indentation and parenthesis setup that made the lint come clear. How should we go about this?